### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3:Table data Text (Pale color) under "Call Tree" tab of "Allocations" Profiler and "Call Tree/Samples" tab of "Time Profiler" are not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#420]
### Steps to Reproduce

Scenario 1:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations" Profiling page. Navigate to "Call Tree" tab and invoke it.
4. Verify the color contrast ratio of Table data (Pale color) text.

Scenario 2:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Time Profiler" Profiling page.
4. Verify the color contrast ratio of Table data (Pale color) text under Call Tree/ Samples tabs

### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 1.8:1 - 2.2:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshots
![12a call tree](https://user-images.githubusercontent.com/22956130/41594463-24439a82-73e1-11e8-968d-e1d03e87a137.png)
![12b call tree](https://user-images.githubusercontent.com/22956130/41594475-2c56890a-73e1-11e8-98e5-8bd0d4d7e80d.png)
![12a samples](https://user-images.githubusercontent.com/22956130/41594490-33d3175c-73e1-11e8-85d3-d5828db87c15.png)
![12b samples](https://user-images.githubusercontent.com/22956130/41594502-3bf08cbc-73e1-11e8-9deb-68f941073db8.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400
